### PR TITLE
rewrite typescript definition file

### PR DESCRIPTION
**Pull request by septs**
The original typescript definition is an error implementation.

### CODE REVIEW

**Comment by rubenv** [comment]
@septs this is a big change. Could you explain a bit what was wrong and why it was fixed like this?

It's hard for me to evaluate such a big rewrite. Usually it's better to work incrementally, solve one problem at a time.

**Comment by septs** [comment]
## The typescript definition
I think there are two classes and two functions under the module.
**PO**, **Item**, **parse**, **load**

## Actually their relationship is
```
default export PO
PO.Item
PO.parse
PO.parsePluralForms
PO.load
```

So the old implementation is completely wrong, I can only use `as any`.

**Comment by rubenv** [comment]
Okay, that makes sense. When I switch diff mode to "ignore whitespace" it becomes much clearer as well, so I see what you did there.

File: pofile.d.ts [hunk 1]
```diff
@@ -1,44 +1,46 @@
-declare module pofile {
-    function parse(data: string): PO;
-    function load(fileName: string, callback: (err: NodeJS.ErrnoException | null, po: PO) => void): void;
+// tslint:disable
```
Comment (rubenv) [inline pofile.d.ts]: Why the disable?

Comment (septs) [reply pofile.d.ts]: removed the line

**Comment by rubenv** [comment]
Ok, good enough, releasing.

Thanks a lot!

**Comment by rubenv** [comment]
Released to NPM as `v1.1.0`.

**Pull request merged**